spec/controllers/documents/intro_controller_spec.rb

```diff
@@ -13,7 +13,7 @@
       end
     end
 
-    context "when there are no document_types_definitely_needed" do
+    context "when no documents have been provided but are needed" do
```

Comment:
This test description doesn't seem right to me. I see there are some documents provided - the `create :document` lines. Can you explain?

Reply:
Or if you revert this change, fine to merge.